bin/crowbar_batch

```diff
@@ -458,7 +458,7 @@ def export_barclamp(barclamp, alias_by_host)
     end
   end
 
-  proposals = get_proposal_list(barclamp)
+  proposals = get_proposal_list(barclamp)[barclamp]
```

Comment:
Yeah but this looks like an unwanted change of the API. Currently without accessing my codebase i have no idea where this have been introduced.